src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -72,7 +77,9 @@ internal interface ISyntaxFactsService : ILanguageService
 
         bool IsRightSideOfQualifiedName(SyntaxNode node);
 
-        bool IsMemberAccessExpressionName(SyntaxNode node);
```

Comment:
There's are 7 methods called `IsMemberAccessExpressionName` in the code base. Why change the name of this one and make it inconsistent?